Template/Domain/Domain.csproj

```diff
@@ -10,19 +10,19 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.4" />
```

Comment:
You have update .net core to V2.1.4. in case of merging this PR, we need to update Olive too

Reply:
Then I'll do it too.
Because dotnetcore v 2.1.2 causes serious problems for one of our users.